k8s/pkg/apis/longhorn/v1beta2/snapshot.go

```diff
@@ -4,6 +4,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type SnapshotHashStatus string
+
+const (
+	SnapshotHashStatusNew        = SnapshotHashStatus("")
+	SnapshotHashStatusInProgress = SnapshotHashStatus("in_progress")
+	SnapshotHashStatusCompleted  = SnapshotHashStatus("complete")
```

Comment:
nit: I know this follows the current convention, but ideally we should use adj word for status/state like`completed` except compound word or noun.

Reply:
Actually, there are many progress status sets in longhorn-manager, longhorn-engine and other packages. I think we can unify them in the future. However, the compatibility will be an issue...

Reply:
Exactly. Let's create a ticket for the future refactoring, not a priority for now.